documentation/book/con-considerations-for-data-storage.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// assembly-overview.adoc
+
+[id='considerations-for-data-storage-{context}']
+
+= Data storage considerations
+
+An efficient data storage infrastructure is essential to the optimal performance of {ProductName}.
+
+{ProductName} requires block storage and works well with cloud-based block storage solutions, including Amazon Elastic Block Store (EBS). The use of file storage is not recommended.
```

Comment:
```suggestion
{ProductName} requires block storage and works well with cloud-based block storage solutions, such as Amazon Elastic Block Store (EBS). The use of file storage is not recommended.
```

I'm not a native speaker. But _including Amazon EBS_ sounds to me as if that is not one of the cloud based storage solutions.

Reply:
Changed